cmd/vela-worker/register.go

```diff
@@ -46,12 +47,16 @@ func (w *Worker) checkIn(config *library.Worker) (bool, string, error) {
 func (w *Worker) register(config *library.Worker) (bool, string, error) {
 	logrus.Infof("worker %s not found, registering it with the server", config.GetHostname())
 
+	config.SetStatus(constants.WorkerStatusIdle)
+
 	tkn, _, err := w.VelaClient.Worker.Add(config)
 	if err != nil {
 		// log the error instead of returning so the operation doesn't block worker deployment
 		return false, "", fmt.Errorf("unable to register worker %s with the server: %w", config.GetHostname(), err)
 	}
 
+	logrus.Infof("worker %s status updated successfully to %s", config.GetHostname(), config.GetStatus())
```

Comment:
```suggestion
	logrus.Infof("worker %q status updated successfully to %s", config.GetHostname(), config.GetStatus())
```
minor, but this will add quotations around the worker name. prevents cases of "worker worker status udpated successfully to idle".. it should out put "worker 'worker' status udpated successfully to idle" instead